### ISSUE TITLE
Question about batch size > 1

### ISSUE THREAD

**Issue by tongyf2333** [xiaomi-research/dggt#35]
In the training code, it seems that the case where the batch size is greater than 1 is not handled. When the batch size > 1, during the masking process, different batches may end up with different numbers of static and dynamic Gaussians after masking. In the actual training process, how do you handle this situation?

**Comment by Ziyi-Xiong1** [comment]
Yes, our current training is conducted with a batch size of 1.
